JS/Lesson34/goksel/recipe-app/src/modules/recipes/NewRecipe/NewRecipeForm/newRecipeHelper.ts

```diff
@@ -0,0 +1,192 @@
+
+export const MealCategory = [
+    {
+        title: "Breakfast",
+        id: "breakfast"
+    },
+    {
+        title: "Chicken",
+        id: "chicken"
+    },
+    {
+        title: "Dessert",
+        id: "dessert"
+    },
+    {
+        title: "Beef",
+        id: "beef"
+    },
+    {
+        title: "Goat",
+        id: "goat"
+    },
+    {
+        title: "Lamb",
+        id: "lamb"
+    },
+    {
+        title: "Miscellaneous",
+        id: "miscellaneous"
+    },
+    {
+        title: "Pasta",
+        id: "pasta"
+    },
+    {
+        title: "Pork",
+        id: "pork"
+    },
+    {
+        title: "Seafood",
+        id: "seafood"
+    },
+    {
+        title: "Side",
+        id: "side"
+    },
+    {
+        title: "Starter",
+        id: "starter"
+    },
+    {
+        title: "Vegan",
+        id: "vegan"
+    },
+    {
+        title: "Vegetarian",
+        id: "vegetarian"
+    }
+]
+
+
+export const MealArea = [
```

Comment:
Same here, it would be good to make Area an enum

Reply:
I used the enum Categories and enum Areas that ı already wrote in models file.